packages/console/src/pages/SignInExperience/components/ColorForm.tsx

```diff
@@ -24,12 +24,17 @@ const ColorForm = () => {
 
   const isDarkModeEnabled = watch('color.isDarkModeEnabled');
   const primaryColor = watch('color.primaryColor');
+  const darkPrimaryColor = watch('color.darkPrimaryColor');
 
   const handleResetColor = useCallback(() => {
     const darkPrimaryColor = absoluteLighten(color(primaryColor), 10);
     setValue('color.darkPrimaryColor', darkPrimaryColor.hex());
   }, [primaryColor, setValue]);
 
+  const showReset = useMemo(() => {
+    return absoluteLighten(color(primaryColor), 10).hex() !== darkPrimaryColor;
```

Comment:
How about useMemo for the generated color result? Then we can reuse it in reset as well